client/webserver/site/src/js/markets.js

```diff
@@ -299,7 +299,7 @@ export default class MarketsPage extends BasePage {
 
       page.marketChart.style.height = `${h}px`
 
-      this.chart.resize(h)
+      this.chart.resize() // resize gets height from parent (marketChart)
```

Comment:
```javascript
console.log("style.height:", page.marketChart.style.height, ", clientHeight:", page.marketChart.clientHeight)
```

When you set the `style.height`, the 'clientHeight' is not affected until the next render or something. This is especially a problem on reload after the user has resized the chart. Their setting is now effectively forgotten, and the chart is the default size again.

Just two steps to repro:
1. Resize chart
2. Refresh

Reply:
Ahhh.  This also worked with `resize_(h)`.  That a solution?

Reply:
That would work, but the trailing `_` is kinda like a "private" method.

Is that better than #1152?

Reply:
#1152 leaves the orphaned event handlers bound because the unbind doesn't match the bind anymore.  If you click back and forth between pages, you end up with many bound handlers and only the latest one bound actually has a valid parent.  If you put a console.log above the `if (!parentHeight)` you'll see the count of logged zeros grows.

Reply:
Ah. Fixed that now.